### PR TITLE
Dont use the Managed Parser on Mac

### PR DESCRIPTION
**Pull request by dellis1972**
Context https://github.com/xamarin/xamarin-android/issues/1670

### CODE REVIEW

**Comment by dellis1972** [comment]
@mrward This might help with your designer issue. Should be fairly easy to hack in if you are able to repo.

File: src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props.in [hunk 1]
```diff
@@ -11,6 +11,7 @@
 		<AndroidVersionCodePattern Condition=" '$(AndroidUseLegacyVersionCode)' != 'True' And '$(AndroidVersionCodePattern)' == '' ">{abi}{versionCode:D5}</AndroidVersionCodePattern>
 		<AndroidResourceGeneratorTargetName>UpdateGeneratedFiles</AndroidResourceGeneratorTargetName>
 		<AndroidUseAapt2 Condition=" '$(AndroidUseAapt2)' == '' ">False</AndroidUseAapt2>
+		<AndroidUseManagedDesignTimeResourceGenerator Condition=" '$(AndroidUseManagedDesignTimeResourceGenerator)' == '' And '$(OS)' == 'Unix' ">False</AndroidUseManagedDesignTimeResourceGenerator>
```
Comment (jonpryor) [inline src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props.in]: Shouldn't this be `$(OS) != 'Windows_NT'`?

**Comment by mrward** [comment]
Are there any plans to backport this change to d15-8?

**Comment by dellis1972** [comment]
@mrward I'll ask. but we might have missed the window. 
Any progress on resolving the underlying issue in VSforMac? I found that `CoreCompile` under msbuild always returns the correct files (this was not the case in xbuild) so the need to parse the csproj as well for intellisense should not be needed.

**Comment by dellis1972** [comment]
@mrward ok, got permission to cp it over, and its done :)

**Pull request merged**